src/actions/Bluetooth/SensorUpdateActions.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+import { PermissionActions } from '../PermissionActions';
+import BleService from '../../bluetooth/BleService';
+
+export const UPDATE_ACTIONS = {
+  SET_LOG_INTERVAL_ERROR: 'Bluetooth/setLogIntervalError',
+  SET_LOG_INTERVAL_START: 'Bluetooth/setLogIntervalStart',
+  SET_LOG_INTERVAL_SUCCESS: 'Bluetooth/setLogIntervalSuccess',
+  DISABLE_BUTTON_START: 'Bluetooth/disableButtonStart',
+  DISABLE_BUTTON_STOP: 'Bluetooth/disableButtonStop',
+};
+
+const setLogIntervalStart = macAddress => ({
+  type: UPDATE_ACTIONS.SET_LOG_INTERVAL_START,
+  payload: { macAddress },
+});
+const setLogIntervalSuccess = () => ({ type: UPDATE_ACTIONS.SET_LOG_INTERVAL_SUCCESS });
+
+const setLogIntervalError = () => ({ type: UPDATE_ACTIONS.SET_LOG_INTERVAL_ERROR });
+
+const disableButtonStart = macAddress => ({
+  type: UPDATE_ACTIONS.DISABLE_BUTTON_START,
+  payload: { macAddress },
+});
+
+const disableButtonStop = macAddress => ({
+  type: UPDATE_ACTIONS.DISABLE_BUTTON_STOP,
+  payload: { macAddress },
```

Comment:
It IRKS me a bit (I don't know why I thought of irk, but I also had to google it, but what a cool word) that this isn't using the success/error pattern (even though it also IRKS me a bit we aren't doing anything with the error hehe)